lib/jsonlint/rake_task.rb

```diff
@@ -24,7 +30,14 @@ def define_task
       task(name) do
         puts 'Running JsonLint...'
 
-        files_to_check = Rake::FileList.new(paths)
+        YamlLint.logger.level = Logger::DEBUG if debug
```

Comment:
Instead of exposing a boolean for debug, I would suggest exposing the log level directly.